projekcia_obrazu:
  ros__parameters:
    point_cloud_topic: /orpheus_1/sensors/velodyne_1/velodyne_points
    # Base camera topic, should include CameraInfo
    camera_topic: /orpheus_1/sensors/realsense_2
    segmentation_topic: TBD
    output_cloud_name: /algorithms/pcfilter
    segmentation_classes: [void, dirt, sand, grass, tree, pole, water, sky, vehicle, container, asphalt, gravel, building, mulch, rock-bed, log, bicycle, person, fence, bush, sign, rock, bridge, concrete, picnic-table, barrier, puddle, mud, rubble]
    segmentation_codes: [24, 5, 6, 7, 1, 23, 2, 3, 8, 0, 22, 21, 20, 19, 11, 18, 17, 16, 15, 9, 14, 4, 10, 13, 12, -1, -1, -1, -1]
    remove_classes: [asphalt, gravel, grass, dirt]
    output_removed: true
    output_removed_cloud_name: /algorithms/pcfilter_inverse